### PR TITLE
🐛 Fix the stream close bug caused by the REL fetch check

### PR DESCRIPTION
**Pull request by dubdabasoduba**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Resolves [link to issue]

**Engineer Checklist**

- [ ] I have written **Unit tests** for any new feature(s) and edge cases for
      bug fixes
- [ ] I have added documentation for any new feature(s) and configuration
      option(s) on the `README.md`
- [ ] I have run `mvn spotless:check` to check my code follows the project's
      style guide
- [ ] I have run `mvn clean test jacoco:report` to confirm the coverage report
      was generated at `plugins/target/site/jacoco/index.html`
- [ ] I ran `mvn clean package` right before creating this pull request.

### CODE REVIEW

File: plugins/src/main/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecision.java [hunk 1]
```diff
@@ -5,12 +5,7 @@
 
 import java.io.FileReader;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (Rkareko) [inline plugins/src/main/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecision.java:8]: Is the use of wildcard import intentional?

File: plugins/src/main/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecision.java [hunk 2]
```diff
@@ -19,11 +14,7 @@
 import org.apache.http.impl.client.BasicResponseHandler;
 import org.apache.http.util.TextUtils;
 import org.hl7.fhir.instance.model.api.IBaseResource;
-import org.hl7.fhir.r4.model.Bundle;
-import org.hl7.fhir.r4.model.ListResource;
-import org.hl7.fhir.r4.model.Location;
-import org.hl7.fhir.r4.model.OperationOutcome;
-import org.hl7.fhir.r4.model.Resource;
+import org.hl7.fhir.r4.model.*;
```
Comment (Rkareko) [inline plugins/src/main/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecision.java:17]: Is the use of wildcard import intentional?

**Review by Rkareko — APPROVED** [review]
